app/jobs/close_patients_job.rb

```diff
@@ -33,8 +42,41 @@ def perform_batch(patients, monitoring_reason, completed_message: false)
       patient[:monitoring_reason] = monitoring_reason
       patient.save!
 
-      # Send closed email to patient if they are a reporter
-      PatientMailer.closed_email(patient).deliver_later if completed_message && patient.email.present? && patient.self_reporter_or_proxy?
+      # Send closed notification to patient if they are a reporter, the closure reason warrants a close
+      # notification, and if the jurisdiction has opted-in to closed notifications.
+      #
+      # If 'sms texted weblink' or 'sms text-message' is preferred, then an SMS will be sent
+      # If 'e-mailed web link' is preferred, then an email will be sent
+      # If the preferred contact method is not supported for close notifications, then a history item will
+      # be created stating so.
+      if completed_message && patient.self_reporter_or_proxy? && jurisdiction_send_close[patient.jurisdiction_id]
+        contact_method = patient.preferred_contact_method&.downcase
+        if ['sms texted weblink', 'sms text-message'].include? contact_method
+          # Do not enqueue if the contact method is blank
+          if patient.primary_telephone.blank?
```

Comment:
This check is also happening at the start of closed_sms, I think either this check or the one in closed_sms might be redundant